test/test_manifest.json

```diff
@@ -5975,5 +5975,12 @@
            "value": true
          }
        }
+    },
+    {  "id": "issue13003",
+       "file": "pdfs/issue13003.pdf",
```

Comment:
I'm not sure why makeref doesn't fail on the bots, since this file isn't included in anywhere in the diff (neither directly or as a linked test-case). Locally this causes makeref to fail completely for me.

@calixteman Can you please do a follow-up PR adding the missing file?